src/graphql/main/schema.graphql

```diff
@@ -211,6 +212,7 @@ type ConsumerAccount implements Account {
   List the quiz questions of the consumer account
   """
   quiz: [Quiz!]!
+  realtimePrice: RealtimePrice!
```

Comment:
the one at the root query should have an input to select the type?

Reply:
this is part of `ConsumerAccount` so there is no need for input. We use the display currency from user account